repose-aggregator/components/filters/translation/pom.xml

```diff
@@ -87,11 +86,6 @@
             <artifactId>xalan</artifactId>
             <version>2.7.1</version>
         </dependency>
-        <dependency>
```

Comment:
dependency managed doesn't make it our dependency, it just says what one we'll get if we do count on the dependency